schemas/com.getvero/bounced/jsonschema/1-0-0

```diff
@@ -12,25 +12,55 @@
 	"properties": {
 		"bounced_at": {
 			"type": "string",				
-			"format": "date-time"
+			"format": "date-time",
+			"maxLength": 10
```

Comment:
What string could include date and time AND be 10 chars long? The only ISO8601-compatible 10 chars string is `YYYY-MM-DD`, but it is clearly only date.

Reply:
I think the datetime strings are all UNIX timestamps: https://help.getvero.com/articles/setting-up-veros-webhooks

Reply:
* `date-time` means that  it is ISO8601-compatible **string** containing both date and time
* UNIX timestamp is an **integer** containing both date and time

If we're just stringifying UNIX timestamp in SCE (which doesn't make much sense) then it won't have `date-time` as it won't be  ISO8601-compatible. If we're converting it to proper `date-time` then it is impossible to have 10 characters length. If we're converting it to `YYYY-MM-DD` (which also doesn't make sense) then it should be just `date`. All these options are mutually exclusive.

Reply:
Okay, this is a blocker on release @rzats. What is the story here?

Reply:
@BenFradet in case @rzats doesn't get back to us in a timely fashion - would you mind looking in the code and telling us which option "must" be correct?

Reply:
https://github.com/snowplow/iglu-central/commit/5eec605939970b87781ab150fe0ea9f1ee1bb0a8 max-length has been removed from this field in the snowflake branch, which is presumably the correct option.